warehouse/query-core/src/main/java/datawave/query/tables/content/ContentQueryTable.java

```diff
@@ -55,7 +55,9 @@ public class ContentQueryTable extends BaseQueryLogic<Entry<Key,Value>> {
     private static final String ALL = "\u10FFFF";
 
     private int queryThreads = 100;
-    private ScannerFactory scannerFactory;
+
+    ScannerFactory scannerFactory;
```

Comment:
If this is package-private, do we really need the getter/setter since they are only used in a test sharing the same package?